### PR TITLE
fix: Added parameter to wrap modal in navigation page

### PR DESCRIPTION
**Pull request by RLittlesII**
**What kind of change does this PR introduce?**

Allows consumer to wrap modals in navigation pages.

**What is the current behavior?**

#66 Broke this functionality with how we are registering the `NavigationView`.
Resolves: #98 

**What is the new behavior?**

You can now pass a `withNavigationPage` to the `PushModal` function.  This is true by default.

**What might this PR break?**

Nothing that isn't already broken.

### CODE REVIEW

**Comment by glennawatson** [comment]
Might be worth adding some unit tests for the diff section here https://codecov.io/gh/reactiveui/Sextant/pull/103/diff

**Comment by RLittlesII** [comment]
@glennawatson Added tests to verify the flag gets to the `IView` instance.

File: Sample/SextantSample/Views/HomeView.xaml.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public HomeView()
 			this.WhenActivated(disposables =>
             {
 				this.BindCommand(ViewModel, x => x.OpenModal, x => x.FirstModalButton).DisposeWith(disposables);
-                this.BindCommand(ViewModel, x => x.PushPage, x => x.PushPage);
+                this.BindCommand(ViewModel, x => x.PushPage, x => x.PushPage).DisposeWith(disposables);
```
Comment (glennawatson) [inline Sample/SextantSample/Views/HomeView.xaml.cs]: Is the alignment off here or is just github?

Comment (RLittlesII) [reply Sample/SextantSample/Views/HomeView.xaml.cs]: Seems it's just github

**Pull request merged**